core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -566,6 +570,33 @@ private FunctionResolver unix_timestamp() {
     );
   }
 
+  /**
+   * UTC_DATE(). return the current UTC Date in format yyyy-MM-dd
+   */
+  private DefaultFunctionResolver utc_date() {
+    return define(BuiltinFunctionName.UTC_DATE.getName(),
+        implWithProperties(functionProperties
+            -> DateTimeFunction.exprUtcDate(functionProperties), DATE));
```

Comment:
Could be simplified
```suggestion
            -> exprUtcDate(functionProperties), DATE));
```
The same for other functions too